optional/fetchmail/fetchmail.py

```diff
@@ -79,7 +61,7 @@ def run(debug):
                 protocol=fetch["protocol"],
                 host=escape_rc_string(fetch["host"]),
                 port=fetch["port"],
-                smtphost=smtphostport if fetch['scan'] else lmtphostport,
+                smtphost=f'{os.environ["FRONT_ADDRESS"]}:25' if fetch['scan'] else f'{os.environ["IMAP_ADDRESS"]}:2525',
```

Comment:
Before `smtphostport` pointed to `SMTP_HOST`

```suggestion
                smtphost=f'{os.environ["SMTP_ADDRESS"]}' if fetch['scan'] else f'{os.environ["IMAP_ADDRESS"]}:2525',
```

Reply:
Done in d5f1f80e ; the syntax with ":" wasn't correct anyways